docs/changelog.md

```diff
@@ -105,6 +105,13 @@ For more information on features and bug fixes in 1.0.0, see the GitHub mileston
 
 #### Upgrade Instructions
 
+##### String vertex ID support
+
+Users now can use custom string vertex ids. See `Custom Vertex ID` chapter in documentation. Prior
```

Comment:
(nitpick) Add link to documentation page:
`[Custom Vertex ID](advanced-topics/custom-vertex-id.md)`

Reply:
👍  Thanks, I forgot we could include link to other doc in changelog